sdk/remoterendering/azure-mixedreality-remoterendering/pom.xml

```diff
@@ -94,6 +94,12 @@
       <version>1.12.0</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.14.2</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <scope>compile</scope>
```

Comment:
We probably only need a test scope here.

Reply:
There is a discussion in the original fix https://github.com/Azure/azure-sdk-for-java/pull/39587 about exactly this, but I don't think we even use netty outside of our tests, so we should be able to use test scope.

Reply:
Discussed offline and just to be safe when also including other projects we'll keep it on compile.

Reply:
Internal communication says `compile` scope and I'm just following other packages by example.